.storybook/webpack.config.js

```diff
@@ -73,5 +73,18 @@ module.exports = webpackSettings => {
     })
   );
 
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    use: [
+      {
+        loader: require.resolve('babel-loader'),
+        options: {
```

Comment:
We should be able to get this compiling without needing to add an additional babel preset. 

I added a comment that has a commit suggestion that will allow us to remove this `options` object and remove `babel-preset-react-app` from `package.json`.

Reply:
removed options object